aspnet/fundamentals/localization.rst

```diff
@@ -204,6 +204,9 @@ CookieRequestCultureProvider
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Production apps will often provide a mechanism to set the culture with the ``ASPNET_CULTURE`` cookie. The sample app provides code to set the cookie for French and English cultures.
```

Comment:
@danroth27 , @JunTaoLuo please review latest change. If ASPNET_CULTURE is not required at all then let me know, I will remove it.